include/zenoh-pico/config.h

```diff
@@ -222,7 +222,7 @@
  */
 #ifndef Z_BATCH_SIZE_RX
 #define Z_BATCH_SIZE_RX \
-    65535  // Warning: changing this value can break the communication
+    3072  // Warning: changing this value can break the communication
```

Comment:
I do not intend to merge changes made to this file. They are just to make the session opening work as my microcontroller doesn't have enough RAM for the default value.